extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import {IntersectionObserver} from '../../../src/intersection-observer';
+import {isPositionFixed} from '../../amp-ad/0.1/amp-ad-3p-impl';
```

Comment:
Not sure what's the question is exactly about. Answer depends on where "is fixed" flag is needed. If it's in the `onLayoutMeasure` or `layoutCallback`, then Resource calculates and maintains `isFixed()` property which can be easily propagated to the custom element.

However, I believe the situation here is more complicated. The Ad maintains its own whitelist of when `position:fixed` is actually allowed. Thus, I don't think we can fully rely on it. This looks like we may have to move this function to `src/ad-helper.js` or such to be compilable into both `amp-ad` and `amp-iframe`. And we can come back and see if we can optimize it with `Resource.isFixed()` at that point. My sense is that it'd be possible.